packages/create-amplify/src/get_project_root.test.ts

```diff
@@ -59,16 +59,18 @@ void describe('getProjectRoot', () => {
 
   void it('creates the project root directory if the user provided relative path does not exist', async () => {
     process.env.npm_config_yes = 'false';
-    const userInput = 'test';
-    mock.method(AmplifyPrompter, 'input', () => Promise.resolve(userInput));
+    const userInputs = ['test', '/test', '/tests/test'];
```

Comment:
These additional test cases shouldn't be in "creates the project root directory if the user provided relative path does not exist". Instead they should be in a test for absolute paths